VideoApi/Testing.Common/Helper/Builders/Domain/ConferenceBuilder.cs

```diff
@@ -182,7 +182,7 @@ public ConferenceBuilder WithInterpreterRoom()
             var room = new Builder(_builderSettings).CreateNew<ParticipantRoom>().WithFactory(() =>
                 new ParticipantRoom(_conference.Id, "InterpreterRoom1", VirtualCourtRoomType.Civilian)).Build();
 
-            var nonJudges = _conference.Participants.Where(x => !x.IsJudge()).ToList();
+            var nonJudges = _conference.Participants.Where(x => x is Participant && !((Participant)x).IsJudge()).ToList();
```

Comment:
can the type of "x" be dynamic here? And why is that?

Reply:
It needs to be casted explicitly.